lecli/cli.py

```diff
@@ -20,315 +20,62 @@ def cli():
-@click.option('-n', '--lognick', default=None,
-              help='Nickname of log in config file')
-@click.option('-g', '--loggroup', default=None,
-              help='Name of log group defined in config file')
-@click.option('-f', '--timefrom',
-              help='Time to get events from (unix epoch)', type=int)
-@click.option('-t', '--timeto',
-              help='Time to get events to (unix epoch)', type=int)
-@click.option('--datefrom',
-              help='Date/Time to get events from (ISO-8601 datetime)')
-@click.option('--dateto',
-              help='Date/Time to get events to (ISO-8601 datetime)')
-@click.option('-r', '--relative_range',
-              help='Relative range to query until now (Examples: today, yesterday, '
-                   'last x timeunit: last 2 hours, last 6 weeks etc.')
-def events(logkeys, lognick, loggroup, timefrom, timeto, datefrom, dateto, relative_range):
-    """Get log events"""
-
-    if lognick:
-        logkeys = api_utils.get_named_logkey(lognick)
-    elif loggroup:
-        logkeys = api_utils.get_named_logkey_group(loggroup)
-
-    if all([logkeys, timefrom, timeto]):
-        query_api.get_events(logkeys, time_from=timefrom, time_to=timeto)
-    elif all([logkeys, datefrom, dateto]):
-        query_api.get_events(logkeys, date_from=datefrom, date_to=dateto)
-    elif all([logkeys, relative_range]):
-        query_api.get_events(logkeys, time_range=relative_range)
-    else:
-        click.echo("Example usage: lecli events 12345678-aaaa-bbbb-1234-1234cb123456 "
-                   "-f 1465370400 -t 1465370500")
-        click.echo("Example usage: lecli events 12345678-aaaa-bbbb-1234-1234cb123456 "
-                   "--datefrom '2016-05-18 11:04:00' --dateto '2016-05-18 11:09:59' ")
-        click.echo("Example usage: lecli events --loggroup myloggroup "
-                   "--datefrom '2016-05-18 11:04:00' --dateto '2016-05-18 11:09:59' ")
-        click.echo("Example usage: lecli events --lognick mynicknamedlog "
-                   "--datefrom '2016-05-18 11:04:00' --dateto '2016-05-18 11:09:59' ")
-        click.echo("Example usage: lecli events --lognick mynicknamedlog "
-                   "-r 'last 3 hours'")
-
-
-@cli.command()
-@click.argument('logkeys', type=click.STRING, nargs=-1)
-@click.option('-n', '--lognick', default=None,
-              help='Nickname of log in config file')
-@click.option('-g', '--loggroup', default=None,
-              help='Name of log group defined in config file')
-@click.option('-l', '--last', default=1200,
-              help='Time window from now to now-X in seconds over which events will be returned '
-                   '(Defaults to 20 mins)')
-@click.option('-r', '--relative_range',
-              help='Relative range to query until now (Examples: today, yesterday, '
-                   'last x timeunit: last 2 hours, last 6 weeks etc.')
-def recentevents(logkeys, lognick, loggroup, last, relative_range):
```

Comment:
We can totally do that.

The problem with this is that there doesn't appear to be a way to do this without the deprecated commands appearing in the help info, and I know that was something we wanted to avoid.

I'm perfectly happy to add it back in if we don't mind the help being a bit cluttered while we support the deprecated commands.